Themes/default/css/index.css

```diff
@@ -1701,7 +1701,7 @@ tr.windowbg td, tr.windowbg2 td, .table_grid tr td {
 .generic_icons.general, .generic_icons.boards, .generic_icons.topics_views {
 	background-position: -31px -31px;
 }
-.generic_icons.gender_Male {
+.generic_icons.gender_1 {
```

Comment:
If no changes are made to the logic where None is skipped because it is a special value then this should be changed from `.generic_icons.gender_1` to `.generic_icons.gender_0`.